cmd/soroban-cli/src/commands/contract/deploy/wasm.rs

```diff
@@ -128,7 +120,13 @@ impl Cmd {
         match res {
             TxnEnvelopeResult::TxnEnvelope(tx) => println!("{}", tx.to_xdr_base64(Limits::none())?),
             TxnEnvelopeResult::Res(contract) => {
-                self.save_contract_id(&contract)?;
+                alias::Data::save_contract_id(
```

Comment:
Given the dependency on `self.config`, then this could become. `self.config.save_contract_id(&alias, &contract)?;`

Reply:
I moved everything to self.config. Should I leave all functions in alias, or actually move them to mod.rs?